src/main/resources/view/DarkTheme.css

```diff
@@ -350,3 +350,17 @@
     -fx-background-radius: 2;
     -fx-font-size: 11;
 }
+
+#moduleTags {
+    -fx-hgap: 7;
+    -fx-vgap: 3;
+}
+
+#moduleTags .label {
+    -fx-text-fill: white;
+    -fx-background-color: #3e7b91;
+    -fx-padding: 1 3 1 3;
+    -fx-border-radius: 2;
+    -fx-background-radius: 2;
+    -fx-font-size: 11;
+}
```

Comment:
Copy pasted from Group Tags. I was thinking to use a different colour for common modules (to make them stand out more).